autogpt_platform/backend/migrations/20241007175112_add_oauth_creds_user_trigger/migration.sql

```diff
@@ -0,0 +1,20 @@
+--CreateFunction
+CREATE OR REPLACE FUNCTION add_user_to_platform() RETURNS TRIGGER AS $$
+BEGIN
+    INSERT INTO platform."User" (id, email, "updatedAt")
+    VALUES (NEW.id, NEW.email, now());
+    RETURN NEW;
+END;
+$$ LANGUAGE plpgsql SECURITY DEFINER;
+
+-- CreateSchema & CreateTable
+-- Note: these CREATE commands are dev-only, where full-fledged Supabase can be missing.
+CREATE SCHEMA IF NOT EXISTS auth;
```

Comment:
@majdyz why is this added?

Reply:
If you only run supabase db for docker, this table does not even exist.